src/styles/index.scss

```diff
@@ -1,3 +1,126 @@
 body {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+}
+
+p {
+  margin: 0;
+  padding: 0;
+}
+
+article {
+  display: inline-block;
+}
+
+.main-container {
+  width: 166px;
+  padding: 32px 15px 16px;
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  align-items: center;
+  border: 1px solid rgb(221, 218, 218);
+  border-radius: 5px;
+}
+
+.main-container__product-img {
+  max-height: 134px;
+  max-width: 160px;
+  display: flex;
+  margin-bottom: 35px;
+}
+
+.main-container__description {
+  display: flex;
+  flex-direction: column;
+  text-align: left;
+}
+
+.main-container__rating-of-product {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  width: 166px;
+  margin-bottom: 34px;
+}
+
+.main-container__product-price {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  width: 166px;
+  height: min-content;
+  margin-bottom: 16px;
+}
+
+.main-container__stars-star {
+  height: 16px;
+  width: 16px;
+  background-image: url(/src/images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.main-container__stars--4 :nth-child(-n + 4) {
+  background-image: url(/src/images/star-active.svg);
+}
+
+.main-container__stars {
+  display: flex;
+}
+
+.main-container__button-buy {
+  height: 38px;
+  width: 164px;
+  text-align: center;
+  background-color: rgba(0, 172, 220, 1);
+  border: 1px solid rgba(0, 172, 220, 1);
+  border-radius: 5px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.main-container__button-buy:hover {
+  background-color: #fff;
```

Comment:
use variables for colors, and fonts. Use hex, don't use named variants as 'white' for example.